iree/vm/test/arithmetic_ops_i64.mlir

```diff
@@ -1,10 +1,22 @@
 vm.module @arithmetic_ops_i64 {
 
   //===--------------------------------------------------------------------===//
-  // I64 Arithmetic
+  // ExtI64: Native integer arithmetic
   //===--------------------------------------------------------------------===//
 
+  // TODO: The CModuleTarget enforces exports to be ordered.
```

Comment:
can we have a tracking issue for this to chat about it?
(for example we may be able to add a pass you could run to reorder if you need them to be in a certain order)

Reply:
We have an internal issue for what we have left to do regarding EmitC. And yeah, as the bytecode module uses `llvm::sort` 
https://github.com/google/iree/blob/96cdbe372cc01a05408a39f1a86e17be51c09ffb/iree/compiler/Dialect/VM/Target/Bytecode/BytecodeModuleTarget.cpp#L92-L101
So we want to make sure the C module behaves similar to the bytecode module.